lib/rollup/index.js

```diff
@@ -1,9 +1,7 @@
-const { join, normalize } = require('path')
+const { dirname, join, normalize } = require('path')
 
 const { rollup } = require('rollup')
-const commonjs = require('rollup-plugin-commonjs')
-const nodeResolve = require('rollup-plugin-node-resolve')
-const { minify } = require('terser')
+const config = require('../../rollup.config')
```

Comment:
Looks like Rollup has a [helper method for programmatically loading a config file](https://rollupjs.org/javascript-api/#programmatically-loading-a-config-file) – should we consider using it here?

Reply:
@36degrees Sadly I had to remove it

The tests crashed when hitting the `loadConfigFile()` call to `await import()`

[Jest doesn't support ESM](https://jestjs.io/docs/ecmascript-modules) without the Node.js `--experimental-vm-modules` flag